cmd/juju/cloud/updatecredential.go

```diff
@@ -35,16 +35,9 @@ a model was created with to the new and valid details on controller.
 This command allows to update an existing, already-stored, named,
 cloud-specific credential on a controller as well as the one from this client.
 
-If a current controller can be detected, a user will be prompted to confirm 
-if specified cloud needs to be updated on it. 
-If the prompt is not needed and the cloud is always to be updated on
-the current controller if that controller is detected, use --no-prompt option.
+Use --controller option to update a cloud on a controller. 
 
-Use --controller option to update a cloud on a different controller. 
-
-Use --controller-only option to only update controller copy of the cloud.
-
-If --client-only is used, Juju updates credential only on this client.
+If --client is used, Juju updates credential on this client.
```

Comment:
This wording differs from elsewhere

Reply:
yes, some original wording makes command help unique. The same is on k8s commands. I've updated what I needed here.